File: Android/FireClassification/app/src/main/java/com/thor/fireclassification/model/DataItem.java
package com.thor.fireclassification.model;

public class DataItem{
	private String date;
	private String imgUrl;
	private double probability;
	private String latitude;
	private String longtitude;
	private String class_name;

	public DataItem(String date, String imgUrl, double probability, String latitude, String longtitude, String class_name) {
		this.date = date;
		this.imgUrl = imgUrl;
		this.probability = probability;
		this.latitude = latitude;
		this.longtitude = longtitude;
		this.class_name = class_name;
	}

	public String getDate(){
		return date;
	}

	public String getImgUrl(){
		return imgUrl;
	}

	public double getProbability(){
		return probability;
	}

	public String getLatitude(){
		return latitude;
	}

	public String getLongtitude(){
		return longtitude;
	}

	public String getClass_name(){
		return class_name;
	}
}

File: Android/FireApp/app/src/main/java/com/thor/firedetectionapp/model/DataItem.kt
package com.thor.firedetectionapp.model


import com.google.gson.annotations.SerializedName

data class DataItem(
    @SerializedName("class_name")
    val className: String,
    @SerializedName("date")
    val date: String,
    @SerializedName("imgUrl")
    val imgUrl: String,
    @SerializedName("latitude")
    val latitude: String,
    @SerializedName("longtitude")
    val longtitude: String,
    @SerializedName("probability")
    val probability: Double



)